src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -354,16 +317,14 @@ FiatExchangeAmount.navOptions = ({
 }) => ({
   ...emptyHeader,
   headerLeft: () => <BackButton eventName={FiatExchangeEvents.cico_add_funds_amount_back} />,
-  headerTitle: () =>
-    route.params?.isCashIn ? (
-      <HeaderTitleWithSubtitle
-        title={i18n.t('addFundsHeaderWithCurrency', {
-          token: route.params.currency === Currency.Celo ? 'CELO' : route.params.currency,
-        })}
-      />
-    ) : (
-      <HeaderTitleWithBalance title={i18n.t('cashOut')} token={route.params.currency} />
-    ),
+  headerTitle: () => (
+    <HeaderTitleWithBalance
+      title={i18n.t(`fiatExchangeFlow.${route.params.flow}.exchangeAmountTitle`, {
```

Comment:
Though this works, having a dynamic string will make it more difficult to switch to extracting i18n strings or validate keys statically. For instance https://www.i18next.com/how-to/extracting-translations#2-using-an-extraction-tool
Could you use a `switch` and static strings instead?